docker-compose.dev.yml

```diff
@@ -2,10 +2,9 @@ version: "3.2"
 
 services:
   postgres:
-    image: postgres:10.0
-    hostname: postgres
+    image: postgres:14
     ports:
-      - "127.0.0.1:15432:5432"
+      - "127.0.0.1:5432:5432"
```

Comment:
I’m on macOS, but I’ve seen some people using certain Linux distros run into problems when using default ports, because they’d also have for example a postgres server running on their host machine already by default. This might be the reason why it was using the non-default ports.

Reply:
I don't think a lot of linux distro's have installed postgres by default. And if people are running postgres without them knowing then making them conscious of that is not a bad thing right?

Reply:
I'm ok with the port change, even if I find it unnecessary.

Reply:
I like the idea of utilizing standard ports as much as possible.